package.json

```diff
@@ -16,8 +16,8 @@
   "scripts": {
     "component": "node -r @babel/register ./build/component.js > ./dist/component.json",
     "compile": "babel src/ -d dist/",
-    "depcheck": "node ./bin/depcheck.js --ignore-dirs=fake_modules",
```

Comment:
We have a nifty [file for this](https://github.com/depcheck/depcheck/blob/master/false-alert.js), you can even add a comment as to why you are adding the import